Adding new concepts and procedures for Multi CV hosts and AKs (#3609)
theforeman/foreman-documentation: guides/common/modules/proc_viewing-content-view-environment-hosts.adoc

Reviewer: Again, I'm not sure if users need something this trivial documented. Leaving to your judgement. Non-blocking.

[id="viewing-content-view-environment-hosts"]
= Viewing content view environment hosts

> Use this procedure to view hosts with multiple content view environments, or multi-environment hosts, in the {ProjectWebUI} and Hammer CLI.

Reviewer: ```suggestion
You can view hosts with multiple content view environments, or multi-environment hosts, in the {ProjectWebUI} and Hammer CLI.
```

.Prerequisites

> Ensure that you have at least one multi-environment host.

Reviewer: ```suggestion
* Ensure that you have at least one multi-environment host.
```
